pinc/ProjectSearchResults.inc

```diff
@@ -350,14 +351,15 @@ class OptionsColumn extends Column
             if ($project->state == PROJ_POST_FIRST_UNAVAILABLE ||
                 $project->state == PROJ_POST_FIRST_AVAILABLE ||
                 $project->state == PROJ_POST_FIRST_CHECKED_OUT) {
-                $actions .= "<a href=\"$projects_url/$projectid/$projectid.zip\">" . _("Download") . "</a><br>\n";
+                $actions[] = "<a href=\"$project->url/$projectid.zip\">" . _("Download") . "</a>";
             }
             if ($project->state == PROJ_POST_SECOND_CHECKED_OUT ||
                 $project->state == PROJ_POST_COMPLETE) {
-                $actions .= "<a href=\"$projects_url/$projectid/".$projectid."_second.zip\">" . _("Download") . "</a><br>\n";
+                $actions[] = "<a href=\"$project->url/".$projectid."_second.zip\">" . _("Download") . "</a>";
```

Comment:
*Very* picky, I know, but, spaces around the dots?

Reply:
Fixed by integrating the variable into the string itself.